### ISSUE TITLE
cx_Oracle is now on GitHub

### ISSUE THREAD

**Issue by anthony-tuininga** [Bluehorn/cx_Oracle#2]
Hi, just a note to let you know that this repository is very much out of date and that cx_Oracle is now on GitHub itself and can be found here: https://github.com/oracle/python-cx_Oracle. Perhaps this repository can be replaced with a fork of the official one?

**Comment by Bluehorn** [comment]
> cx_Oracle is now on GitHub itself

Yep, I noticed as much (after mirroring the Subversion repository here and later forking the Mercurial repository on Bitbucket...).
Unfortunately I can't just replace the whole repository because we still have builds relying on the local patches. And for unknown reasons any newer, official cx_Oracle release cripples the performance of our application for basically trivial queries (single table, queried by primary key).

I am not sure if I should open a bug since I can not reproduce it on any other Oracle installation but customer production systems.

**Comment by anthony-tuininga** [comment]
Can you provide the local patches you require? That might help determine if those patches ought to be part  of the main repository. You can send them directly to me instead of posting them here, if that helps.

**Comment by Bluehorn** [comment]
> Can you provide the local patches you require?

It's only one: https://github.com/Bluehorn/cx_Oracle/commit/db38a7aee5180b3549499f80e7899055a9321410

And it is not needed for newer versions of cx_Oracle AFAIK. Even if the change still had any effects, it is not relevant to us anymore as SQLAlchemy now works around this. See

https://bitbucket.org/zzzeek/sqlalchemy/issues/2561

I updated the description of this repository to link to the official one for now. I will delete this repository when I am sure that we are not going to need this for any builds.

**Comment by anthony-tuininga** [comment]
Thanks for the explanation.
